atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -142,6 +143,13 @@ def get_output(self, worker_metadatas):
 
         return outputs
 
+    def get_parent_image_koji_build_id(self):
+        res = self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY)
+        if res is None:
+            return
+        res = self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY, {})
```

Comment:
Why compute `res` again?

If you're passing a default param to `get`, `{}`, might as well just change this whole method to:
```python
def get_parent_image_koji_build_id(self):
  return self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY, {}).get('parent-image-koji-build-id')
```